pyproject.toml

```diff
@@ -28,10 +28,11 @@ readme = 'README.md'
 version = '0.1.0'
 
 [tool.poetry.dependencies]
-pyfiglet = '>=0.8.0'
 python = '>=3.8'
-setuptools = '69.0.3'
+setuptools = '75.1.0'
 toml = '>=0.10.2'
+certifi = '2024.07.04'
+idna = '3.7'
```

Comment:
If these are transitive dependencies, I feel that we shouldn't have to add them ourselves explicitly?